wandb/sdk/data_types/video.py

```diff
@@ -100,6 +102,11 @@ def __init__(
                 "wandb.Video accepts {} formats".format(", ".join(Video.EXTS))
             )
 
+        if isinstance(data_or_path, (BytesIO, str)) and fps:
+            wandb.termwarn(
+                "`fps` argument does not affect the frame rate of the video when providing raw bytes."
```

Comment:
or a file path?

Reply:
good catch, missed adding that part to the warning message.